hr_language/models/hr_language.py

```diff
@@ -30,12 +29,36 @@ class hr_language(models.Model):
         tools.scan_languages(),
         string=u"Language", required=True)
     description = fields.Char(
-        string=u"Description", size=64, required=True)
+        string=u"Description", size=64, required=False)
```

Comment:
Put a comment for reminding that this field has to be removed when migrating to next version.